File: evo.funders.commonmessages/v1/PHP/src/Request/OpenBankingRequest.php
<?php


namespace AzureFunderCommonMessages\PHP\Request;


use AzureFunderCommonMessages\PHP\Traits\JsonSerializer;

class OpenBankingRequest extends AbstractData implements \JsonSerializable
{
    use JsonSerializer;

    private string $funderApplicationId;

    /**
     * @return string
     */
    public function getFunderApplicationId(): string
    {
        return $this->funderApplicationId;
    }

    /**
     * @param string $funderApplicationId
     * @return OpenBankingRequest
     */
    public function setFunderApplicationId(string $funderApplicationId): OpenBankingRequest
    {
        $this->funderApplicationId = $funderApplicationId;
        return $this;
    }

    public static function fromJson(array $data): self
    {
        $instance = new self();
        $instance->funderApplicationId = $data["funderApplicationId"];
    }

}

File: evo.funders.commonmessages/v1/DotNet/Request/OpenBankingRequest.cs
using AzureFunderCommonMessages.DotNet.Request.DataTypes;
using Newtonsoft.Json;

namespace AzureFunderCommonMessages.DotNet.Request
{
    public class OpenBankingRequest : BaseCommonRequest
    {
        public OpenBankingRequest()
        {
            Data = new OpenBankingRequestData();
        }
        
        [JsonProperty("data", Required = Required.Always)]
        public new OpenBankingRequestData Data { get; set; }
    }
}